pom.xml

```diff
@@ -127,11 +127,14 @@
 
       <plugin>
         <artifactId>maven-jar-plugin</artifactId>
-        <version>2.3.2</version>
+        <version>3.2.0</version>
         <configuration>
           <!-- Pick the MANIFEST generated by the bundle plugin -->
           <archive>
             <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+            <manifestEntries>
+              <Automatic-Module-Name>org.sqlite.jdbc</Automatic-Module-Name>
```

Comment:
This is wrong because this driver is not developed by SQLite devs.

Reply:
You're right, I see. What would you suggest instead? I'm thinking of something like `com.github.xerial.sqlitejdbc`

Reply:
`org.xerial.sqlitejdbc`? Group id plus artifact id.